survey_form/index.html

```diff
@@ -1,134 +1,146 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
     <title>Title of the document</title>
-    <meta name="description" content="The HTML5 Herald">
-    <meta name="author" content="SitePoint">
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <link rel="stylesheet" type="text/css" href="./index.css">
-    <link href="https://fonts.googleapis.com/css?family=Noto+Serif+SC" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Bangers|Charmonman|Monoton" rel="stylesheet">
-</head>
-
-<body>
+    <meta name="description" content="The HTML5 Herald" />
+    <meta name="author" content="SitePoint" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1"
+    />
+    <link rel="stylesheet" type="text/css" href="./index.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Noto+Serif+SC"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Charmonman|Monoton"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
     <header>
-        <div class="navi">
-            <div class="logo">
-                <h2>Zdybel</h2>
-            </div>
-
-
-            <nav>
-
-                <ul>
-                    <li><a href=#>Home</a></li>
-                    <li><a href=#>Contact</a></li>
-                    <li><a href=#>About</a></li>
-                    <li><a href=#>Log In</a></li>
-                </ul>
-
-            </nav>
-        </div>
+      <div class="navi">
```

Comment:
Proponuję wdrożyć konwencję nazewnictwa klas CSS - [BEM](http://getbem.com/): `block__element--modifier`.

BEM zastosowany na tym bloku kodu (trochę zmieniłem strukturę HTML) może wyglądać np. tak:
```
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item nav__item--logo">
            <a href="#" class="nav__link nav__link--logo">Zdybel</a>
          </li>
          <li class="nav__item"><a href="#" class="nav__link">Home</a></li>
          <li class="nav__item"><a href="#" class="nav__link">Contact</a></li>
          <li class="nav__item"><a href="#" class="nav__link">About</a></li>
          <li class="nav__item"><a href="#" class="nav__link">Log In</a></li>
        </ul>
      </nav>
```
Dodatkowe źródło: (BEM)[https://medium.freecodecamp.org/a-quick-introduction-to-block-element-modifiers-bem-9df46d29b64c].

Także następnym krokiem jest zamiana części kodu HTML na powyższy i odpowiednie dostosowanie CSSów, żeby layout zachowywał się tak samo.